src/dos/cdrom_ioctl_linux.cpp

```diff
@@ -113,4 +291,154 @@ bool CDROM_Interface_Ioctl::ReadSectorsHost([[maybe_unused]] void *buffer,
 	return false; /*TODO*/
 }
 
+void CDROM_Interface_Ioctl::InitAudio()
+{
+	// Find a a channel name that doesn't exist (channel names must be
+	// unique or they get overwritten)
+	mixer_channel_t exists;
+	int num                  = 0;
+	const std::string prefix = MixerChannelNamePrefix;
+	std::string full_name;
+	do {
+		full_name = prefix + std::to_string(num++);
+		exists    = MIXER_FindChannel(full_name.c_str());
+	} while (exists);
+
+	audio_buffer = std::make_unique<int16_t[]>(AudioBufferBytes / 2);
+	mixer_channel = MIXER_AddChannel(std::bind(&CDROM_Interface_Ioctl::CDAudioCallback,
```

Comment:
Maybe break the `std::bind` stuff out into a temp var.

Reply:
Fixed